OneCommunityGlobal/HighestGoodNetworkApp
# Xiao update shadow consistency throughout app

# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/68f72cfa-d91d-4296-88e6-5aaf7bb1e483)

## Main changes explained:
Added missing shadow style for buttons throughout app by applying style={boxStyle} to the buttons

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as owner/admin user
4. check that the main buttons throughout the app has shadow style applied to them

## Screenshots or videos of changes:
Other Links -> Projects -> click on the "WBS" button for any project
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/f0d3cd3d-b247-4bfe-981b-fae5a5e2de04)

Other Links -> Projects -> click on the "Members" button for any project -> on the right side of "Find User" search tab, click "All"
![Pasted Graphic 1](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/b749b70d-56a9-4ef2-a911-cb4231cec175)

User Profile -> Change Photo -> update a photo that exceeds 50KB
![Pasted Graphic 3](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/1b820c71-2cc2-4aba-8fa6-f11f7de0e610)

Weekly Summaries -> Move This Summary -> choose a different week
![Pasted Graphic 4](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/d201a79a-bb13-49af-9f1d-1924ff51673d)

Weekly Summaries
![Pasted Graphic 5](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/7fe065d0-9b76-4d9c-92ef-8f4ec038afcb)

Dashboard -> Team Member Tasks -> click on the red bell next to a task
![Pasted Graphic 6](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/2d536619-c5fb-4736-9146-48081b592ab3)

Leaderboard -> click on the dot to the left of the username
![Pasted Graphic 7](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/453685c0-6b70-4344-a515-c241c1d8d7ea)

Dashboard -> Report App Bug
![Pasted Graphic 8](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/96d2106d-fbb5-4ed7-860b-2f426c4a124b)

Leaderboard -> click on the info icon next to “Note: You are currently invisible to the team(s) you are on. ”
![Pasted Graphic 9](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/a6396866-ab1a-4fc7-89f0-c4fd1880ea5c)

Tasks and Timelogs -> Current Week Timelog -> click the trash icon for one of the time entries
![Pasted Graphic 10](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/9cd5ee4e-8eaa-4310-9357-98ccb3ce6bdf)

User Profile -> switch to a smaller screen size
![Pasted Graphic 15](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/63f3544a-6961-48c8-bd3c-6317fd750103)

User Profile -> Basic Information
![Pasted Graphic 11](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/4677192a-7630-4309-90fc-d45678e64d03)

User Profile -> Volunteering Times
![Pasted Graphic 12](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/59a6ce65-2a82-41cb-b01a-3484222e8c94)

User Profile -> Teams
![Pasted Graphic 13](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/ab235bd5-127a-4fdc-8094-075f4d529480)

User Profile -> "Select Featured" button (for badges)
![Pasted Graphic 16](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/eb72dc8f-0310-441c-981e-52ca753d7825)

Log in as Owner -> Permission Management
![Pasted Graphic 18](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/ec7c8c07-550e-4783-b30e-f8d63299ca46)

## Note
Please check throughout the app to make sure the style stays consistent.
In the future, in case you need to add any new buttons for a functionality, please import { boxStyle } from 'styles' and apply the style to the new buttons.
